build-system/tasks/runtime-test/index.js

```diff
@@ -432,9 +451,13 @@ async function runTests() {
    * @param {string} batchName a human readable name for the batch.
    * @param {!Array{string}} browsers list of SauceLabs browsers as
    *     customLaunchers IDs.
+   * @param {!Function} runComplete a function to execute on the `run_complete`
```

Comment:
nit: For clarity, rename this param to `runCompleteFn`. (I was confused until I saw this line.)

Reply:
Done